index.html

```diff
@@ -0,0 +1,120 @@
+</head>
+
+<body>
+
+  <header>
+    <img class="coverImg" src="images/gesture-intro.jpg">
+    <div class="container">
+      <!-- <img class="coverImg" src="images/gesture-intro.jpg"> -->
+      <nav>
+        <ul>
+          <li><a href="#">Support</a></li>
+          <li><a href="#">How To</a></li>
+          <li><a href="#">About</a></li>
+        </ul>
+      </nav>
+      <h1>Gesture Help Desk</h1>
+    </div>
+  </header>
+
+  <div class="main">
+    <section class="about">
+      <h2>About</h2>
+      <div class="grid">
+        <div class="col-1-3">
+          <a class="button" href="#">Who We Are</a>
+          <img src="images/WhoWeAre.png" />
+        </div>
+        <div class="col-1-3">
+          <a class="button" href="#">Auction Basics</a>
+          <img src="images/AuctionBasics.png" />
+        </div>
+        <div class="col-1-3">
+          <a class="button" href="#">Technology Basics</a>
+          <img src="images/Gesture_Logo.jpg" />
+        </div>
+      </div>
+    </section>
+
+    <section class="howTo">
+      <h2>How To</h2>
+      <div class="grid">
+        <div class="col-1-3">
+          <a class="button" href="#">Getting Started</a>
+          <img src="images/GettingStarted.png" />
+        </div>
+        <div class="col-1-3">
+          <a class="button" href="#">Settings</a>
+          <img src="images/Settings.png" />
+        </div>
+        <div class="col-1-3">
+          <a class="button" href="#">Users</a>
+          <img src="images/Users.png" />
+        </div>
+        <div class="col-1-3">
+          <a class="button" href="#">Dashboard</a>
+          <img src="images/Dashboard.png" />
+        </div>
+        <div class="col-1-3">
+          <a class="button" href="#">Displays</a>
+          <img src="images/Displays.png" />
+        </div>
+        <div class="col-1-3">
+          <a class="button" href="#">Check-In</a>
+          <img src="images/CheckIn.png" />
+        </div>
+        <div class="col-1-3">
+          <a class="button" href="#">Checkout</a>
+          <img src="images/CheckOut.png" />
+        </div>
+        <div class="col-1-3">
+          <a class="button" href="#">Ads</a>
+          <img src="images/Ads.png" />
+        </div>
+        <div class="col-1-3">
+          <a class="button" href="#">Messages</a>
+          <img src="images/Messages.png" />
+        </div>
+      </div>
+    </section>
+
+    <section class="support">
+      <h2>Support</h2>
+      <div class="col-1-3">
+        <a class="button" href="#">FAQs</a>
+        <img src="images/FAQs.png" />
+      </div>
+      <div class="col-1-3">
+        <a class="button" href="#">File a Support Ticket</a>
+        <img src="images/FileATicket.png" />
+      </div>
+      <div class="col-1-3">
+        <a class="button" href="#">Contact a Rep</a>
+        <img src="images/ContactARep.png" />
+      </div>
+    </section>
+
+    <footer>
+      <img class="footerLogo" src="images/Gstr_Logo_CMYK_Inline.png">
```

Comment:
Use footer-logo instead of footerLogo. Snake your classes together using a dash instead of camel casing.